### ISSUE TITLE
Suo::LockClientError

### ISSUE THREAD

**Issue by dkdelosreyes** [nickelser/activejob-traffic_control#11]
Hi @nickelser 

This error is happening on production only. Could this be a cause of missing this code below in the configs?

`ActiveJob::TrafficControl.client = ConnectionPool.new(size: 5, timeout: 5) { Redis.new }`

**Comment by richpeck** [comment]
Yes, I had that issue in Heroku and by placing that code into the app it works.

**Comment by corymcdonald** [comment]
Just had this same issue, if you don't have `Redis` running, or defined in your project then you will get this error. It would be nice if this error was more clear, i.e. `CannotConnectError`

**Comment by elioncho** [comment]
Documentation needs to tell that the line above is required.
